README.md

```diff
@@ -1,70 +1,230 @@
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Analyzing the Bundle Size
+<!-- GETTING STARTED -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## 💻 Getting Started <a name="getting-started"></a>
 
-### Making a Progressive Web App
+> Clone the repository by clicking on the 'Code' button and copy the link
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+To get a local copy up and running, follow these steps.
 
-### Advanced Configuration
+### Prerequisites
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+In order to run this project you need:
 
-### Deployment
+-visual studio
+-github
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+### Setup
 
-### `npm run build` fails to minify
+Clone this repository to your desired folder:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+```
+git clone https:https://github.com/edahigure/space-travelers-hub.git
+I run this command
+npx create-react-app my-app  to install react, react-dom, and react-scripts with cra-template...
+```
+
+### Install
+
+- Install all the dependencies required for the website. By using the command below.
+```
+Run npm install to install all dev dependencies for Linters, Webpack, Jest & Babel.
+```
+  
+- Open the index.html file in a browser.
+-
+
+### Install
+
+Install this project with:
+
+```
+  npm install
+```
+### Usage I
+
+Change the main branch to the right one, execute the following command:
+
+```sh
+  git checkout event
+```
+
+### Usage II
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+To run the webhint linter
+
+```sh
+  npx hint .
+```
+
+To run the stylehint linter
+
+```sh
+  npx stylehint "**/*.{css,scss}"
+```
+
+To run the ESLint linter
+
+```sh
+  npx eslint "**/*.{js,jsx}" 
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **TUMBU BARTHOLOMEW ELAD**
+GitHub: [@githubhandle](https://github.com/elad237)
+- Twitter: [@twitterhandle](https://twitter.com/Elad59380989)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tumbu-elad-896ab2183/)
+
+👤 **Name**
+GitHub: [@githubhandle]()
+- Twitter: [@twitterhandle]()
+- LinkedIn: [LinkedIn]()
```

Comment:
- [ ] For this author, the handles are not set. Kindly add links to the second author's social media accounts for ease of contact should potential employers end up looking at this project in the future or any future collaborator, etc. Please add them 👍🏽